stock-tracker/src/loading.css

```diff
@@ -0,0 +1,33 @@
+.lds-ripple {
+  display: flex;
+  margin: 10% auto;
+  position: relative;
+  width: 64px;
+  height: 64px;
+}
+.lds-ripple div {
```

Comment:
This could be done without any extra `<div>` elements, using the `:before` and `:after` CSS pseudo-elements